tasks/release.js

```diff
@@ -202,17 +212,17 @@ function deterministicTar () {
   var extract =
     tar.extract()
       .on('entry', function (header, stream, cb) {
-        if (header.type !== 'file') return cb()
-
         header.mtime = header.atime = header.ctime = UNIXZERO
         header.uid = header.gid = 0
 
         delete header.uname
         delete header.gname
 
-        header.mode = 0o777
```

Comment:
Execution on Linux still works without this?

Reply:
Now we inherit the file mode from the `lstat`, so that we don't have to give full permissions to every file. https://github.com/cosmos/voyager/pull/490/files#diff-d95e7c916be2a81522fee0617e0d16a9R177